salz-core/src/viewer/ViewerDraw.hs

```diff
@@ -23,9 +23,9 @@ drawBoard state = withBorderStyle BS.unicodeBold
   $ vBox rows
   where
     map_ = VS.board state
-    (vx, vy) = VS.location state
+    (Map.C vx vy) = VS.location state
     rows = [hBox $ cellsInRow r | r <- [99,98..0]]
-    cellsInRow y = [drawCoord (Map.toCoord (x+vx, y+vy)) | x <- [0..99]]
+    cellsInRow y = [drawCoord $ Map.C ((Map.I x)+vx) ((Map.I y)+vy) | x <- [0..99]]
```

Comment:
`Map.I` is the constructor for a modular integer, that is because the map wraps around and so 101 is the same thing as 1 if the map size is 100
`Map.C` is the constructor for a coordinate which is just a pair of those modular integers
`Map.toCoord` is an alternative coordinate constructor that takes an int tuple

Checkout the salz-core/src/lib/Map.hs file if you want to investigate how I put them together :)

Reply:
I see. Can I suggest making the code itself more descriptive? Like maybe `Map.I` becomes `Map.ModInt` and `Map.C` becomes `Map.Cell`?

Just to make sure, we're talking about the first index being 1 and not 0, since we're in Haskell, right?

Reply:
Yeah, I'll change the name of those constructors.

Which index are you talking about?

Reply:
Sorry, not sure why I called it index when I meant the origin of the map. Is the origin (0, 0) or (1, 1)?

Reply:
(0, 0) is in the map!

Reply:
Also I changed the Map type constructor names.

Reply:
Alright. I'll mark this as resolved then :)

Reply:
btw, if it's just a comment, you don't need to set to "requesting changes". Once we have CI/CD in, you'll only be able to merge with an approval (or something like that, obviously we need some leeway here and there for those small changes)